.github/workflows/bench.yml

```diff
@@ -78,7 +78,7 @@ jobs:
         github-token: ${{ secrets.GITHUB_TOKEN }}
         # Store the results and deploy GitHub pages automatically if the results are from main branch
         auto-push: ${{ inputs.publish }}
-        comment-on-alert: true
+        comment-on-alert: false
```

Comment:
This is for throughput check. Do we also want to disable them?

Reply:
Reverted changes for throughput benchmarks with https://github.com/awslabs/mountpoint-s3/pull/1063/commits/df9d7c3953a5b21a272314b70cfff7fcd4470f24